eng/spotbugs-aggregate-report/pom.xml

```diff
@@ -23,15 +23,13 @@
 
     <!-- Track 2 client library versions -->
     <azure-core.version>1.0.0-preview.3</azure-core.version>
```

Comment:
should this be -preview.4 since you've updated the version in the pom.xml

Reply:
Yes!